source/documentation/3.9/rspec-mocks/RSpec/Mocks/Syntax.html

```diff
@@ -335,13 +336,13 @@ <h3 class="signature first" id="default_should_syntax_host-class_method">
     <td>
       <pre class="code"><span class="info file"># File 'lib/rspec/mocks/syntax.rb', line 181</span>
 
-<span class='kw'>def</span> <span class='kw'>self</span><span class='period'>.</span><span class='id identifier rubyid_default_should_syntax_host'>default_should_syntax_host</span>
-  <span class='comment'># JRuby 1.7.4 introduces a regression whereby `defined?(::BasicObject) =&gt; nil`
+<span class='kw'>def</span> <span class='kw'>self</span><span class='period'>.</span><span class='id identifier rubyid_default_should_syntax_host'>default_should_syntax_host</span>  <span class='comment'># JRuby 1.7.4 introduces a regression whereby `defined?(::BasicObject) =&gt; nil`
 </span>  <span class='comment'># yet `BasicObject` still exists and patching onto ::Object breaks things
 </span>  <span class='comment'># e.g. SimpleDelegator expectations won&#39;t work
 </span>  <span class='comment'>#
 </span>  <span class='comment'># See: https://github.com/jruby/jruby/issues/814
-</span>  <span class='kw'>if</span> <span class='kw'>defined?</span><span class='lparen'>(</span><span class='const'>JRUBY_VERSION</span><span class='rparen'>)</span> <span class='op'>&amp;&amp;</span> <span class='const'>JRUBY_VERSION</span> <span class='op'>==</span> <span class='tstring'><span class='tstring_beg'>&#39;</span><span class='tstring_content'>1.7.4</span><span class='tstring_end'>&#39;</span></span> <span class='op'>&amp;&amp;</span> <span class='const'>RUBY_VERSION</span><span class='period'>.</span><span class='id identifier rubyid_to_f'>to_f</span> <span class='op'>&gt;</span> <span class='float'>1.8</span>
+</span>
+  <span class='kw'>if</span> <span class='kw'>defined?</span><span class='lparen'>(</span><span class='const'>JRUBY_VERSION</span><span class='rparen'>)</span> <span class='op'>&amp;&amp;</span> <span class='const'>JRUBY_VERSION</span> <span class='op'>==</span> <span class='tstring'><span class='tstring_beg'>&#39;</span><span class='tstring_content'>1.7.4</span><span class='tstring_end'>&#39;</span></span> <span class='op'>&amp;&amp;</span> <span class='const'>RUBY_VERSION</span><span class='period'>.</span><span class='id identifier rubyid_to_f'>to_f</span> <span class='op'>&gt;</span> <span class='float'>1.8</span>
```

Comment:
Is this spacing change from your script?

Reply:
Ok. I rerun the script with the same Ruby version and the diff is simpler.

```diff
--- a/source/documentation/3.9/rspec-mocks/RSpec/Mocks/MessageExpectation.html
+++ b/source/documentation/3.9/rspec-mocks/RSpec/Mocks/MessageExpectation.html
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 <title>
   Class: RSpec::Mocks::MessageExpectation

@@ -9,9 +10,9 @@

 </title>

-  <link rel="stylesheet" href="/documentation/3.9/rspec-mocks/css/style.css" type="text/css" charset="utf-8" />
+  <link rel="stylesheet" href="/documentation/3.9/rspec-mocks/css/style.css" type="text/css" />

-  <link rel="stylesheet" href="/documentation/3.9/rspec-mocks/css/common.css" type="text/css" charset="utf-8" />
+  <link rel="stylesheet" href="/documentation/3.9/rspec-mocks/css/common.css" type="text/css" />

 <script type="text/javascript">
   pathId = "RSpec::Mocks::MessageExpectation";
@@ -1926,22 +1927,11 @@ arguments.</p>
 </div>

       <div id="footer">
-  Generated on Fri Mar  6 15:02:29 2020 by
+  Generated on Wed Apr  8 02:25:46 2020 by
   <a href="http://yardoc.org" title="Yay! A Ruby Documentation Tool" target="_parent">yard</a>
   0.9.24 (ruby-2.6.3).
 </div>

-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-  ga('create', 'UA-59095215-1', 'auto');
-  ga('send', 'pageview');
-
-</script>
-
```

I will jump back to Ruby 2.6.3 for an easier diff.